src/display/annotation_layer.js

```diff
@@ -2949,11 +2972,21 @@ class SquigglyAnnotationElement extends AnnotationElement {
   }
 
   render() {
-    if (!this.data.popupRef && this.hasPopupData) {
+    const {
+      data: { highlightedText, popupRef },
+    } = this;
+    if (!popupRef && this.hasPopupData) {
       this._createPopup();
     }
 
     this.container.classList.add("squigglyAnnotation");
+
+    if (highlightedText) {
+      const underline = document.createElement("u");
+      underline.classList.add("highlightedText");
+      underline.textContent = highlightedText;
+      this.container.append(underline);
+    }
```

Comment:
Let's add a newline after this line given that the other three classes have it as well (alternatively, remove it there too).